src/characters/tDec.ts

```diff
@@ -32,7 +32,7 @@ export async function generateTDecEntities(
   endDate: Date,
   porterUri: string,
   aliceSecretKey: SecretKey = SecretKey.random()
-): Promise<readonly [Enrico, tDecDecrypter, EnactedPolicy]> {
+): Promise<readonly [Enrico, tDecDecrypter, EnactedPolicy, any]> {
```

Comment:
We can avoid using `any` here by implementing `interface TDecConfig` as type for return value